### DISCUSSION TITLE
GitHub copilot group access for university course students

### DISCUSSION THREAD

**Discussion by tomasmta** [community/community#13740] [Copilot Conversations] [answered]
Good morning everyone. 

I am a Master student in National Cheng Kung University, and part of the Nordling Lab. My advisor, Professor Nordling, and I are interested in the possibility of using GitHub Copilot for educational and research purposes. 

My advisor is teaching a information retrieval course. in this course the students will work on developing  a state of the art pipeline for extracting the content of PDF into a structured XML format. 

Does anyone know who could we reach out to get special access for our group? I know we could ask our students to each request for access to Copilot individually, but since our course in built upon the principle of collaborative learning, we would love to give all of our students the same resources. 

Thanks

**Comment by v1ctorio** [comment, marked as answer]
I think instead of using GitHub copilot, you should use [Codex](https://beta.openai.com/docs/guides/code/introduction), the API behind GitHub Copilot. To access the API, you can request it to OpenAI through this form https://beta.openai.com/codex-waitlist.

**Reply by tomasmta** [reply]
Thank you so much for your reply. We applied for codex months ago, but unfortunately have not gotten a response.
